### ISSUE TITLE
Jacobian Analysis: CONVERTD has been dropped since GAMS v.43

### ISSUE THREAD

**Issue by olejandro** [kanors-emr/Veda2.0-Installation#31]
According to the release [notes](https://www.gams.com/43/docs/RN_43.html) `CONVERTD` (CONVERT alias) was dropped in GAMS version 43. Therefore selecting `CONVERTD` as a solver in Veda leads to a GAMS error.

**Comment by olejandro** [comment]
Selecting `CONVERT` instead seems to work. However option `jacobian` appears depreciated.

**Comment by olejandro** [comment]
Apparently option `DumpGDX` replaced option `jacobian` in GAMS version 34, according to the release [notes](https://www.gams.com/34/docs/RN_34.html).

**Comment by olejandro** [comment]
@Antti-L, I agree that it is important to keep this working for users with older GAMS versions.

Release notes state that `CONVERTD` is an alias for `CONVERT`. Won't using `CONVERT` instead of `CONVERTD` in older version not work?

**Comment by olejandro** [comment]
In any case, this error message should probably be updated to provide more accurate instructions (i.e. make them GAMS version specific).
![image](https://github.com/kanors-emr/Veda2.0-Installation/assets/25687793/14e86bb8-39a4-4532-82c7-c05d70540806)

**Comment by Antti-L** [comment]
> Release notes state that CONVERTD is an alias for CONVERT. Won't using CONVERT instead of CONVERTD in older version not work?

CONVERTD is not an alias of CONVERT, but a version of CONVERT that passes information from GAMS to CONVERT **in core** without use of temporary files.  It is designed to speed up GAMS to solver communications.  CONVERTD also has some options that CONVERT does not support, e.g. the localsolver option to convert GAMS models into the LocalSolver language.  If the user would like to use CONVERTD (and not CONVERT), I think it would helpful if VEDA continues to support that.  I am sure GAMS users know which solvers their version includes (because reading the documentation is essential), and keeping it in the list would do no harm to users of new GAMS versions.

**Comment by olejandro** [comment]
Thanks @Antti-L. Tbh, it is new Veda users that I'm most concerned about. I am sure seasoned GAMS and Veda users can find their way arround.

In any case, making the error message more accurate, I believe, would do the trick.
